atoma-proxy/docs/openapi.yml

```diff
@@ -472,6 +477,11 @@ components:
           example: meta-llama/Llama-3.3-70B-Instruct
     ChatCompletionChunkChoice:
       type: object
+      description: |-
+        Represents the chat completion chunk choice.
+
+        This is used to represent the chat completion chunk choice in the chat completion request.
+        It can be either a chat completion chunk delta or a chat completion chunk choice.
```

Comment:
Should we have the properties documented here? the `index` and `delta` are required so at the very least I think they should be

Reply:
I removed this comment, since we are already documenting the chunk delta, etc